test/common.py

```diff
@@ -151,13 +152,18 @@ def safeCoalesce(self, t):
 
         return tg
 
+    def unwrapVariables(self, x, y):
+        if isinstance(x, Variable) and isinstance(y, Variable):
+            return x.data, y.data
+        elif isinstance(x, Variable) or isinstance(y, Variable):
+            raise AssertionError("cannot compare %s and %s" % (type(x), type(y)))
```

Comment:
can you change it to use `.format`? I think `%` for string interpolation is deprecated in Python 3.